daemon/daemon_main.go

```diff
@@ -1552,3 +1570,35 @@ func checkNodePortAndEphemeralPortRanges() error {
 	}
 	return nil
 }
+
+func (d *Daemon) launchHubble() {
+	logger := logging.DefaultLogger.WithField(logfields.LogSubsys, "hubble")
+	addresses := option.Config.HubbleListenAddresses
+	if len(addresses) == 0 {
+		logger.Info("Hubble server is disabled")
+		return
+	}
+	for _, address := range addresses {
+		if !strings.HasPrefix(address, "unix://") {
+			logger.WithField("addresses", addresses).
+				Warn("Hubble server currently only supports listening on Unix domain sockets")
+			return
+		}
+	}
+	payloadParser, _ := parser.New(d, d, d, ipcache.IPIdentityCache, d)
```

Comment:
Why is the error from `parser.New()` ignored here? If not nil, then `payloadParser` will be nil which can cause a panic if used.

Reply:
done